src/epics/screen.ts

```diff
@@ -43,9 +43,10 @@ const processPostInvoke: Epic = (action$, store) => action$.ofType(types.process
             const postInvoke = action.payload.postInvoke as OperationPostInvokeRefreshBc
             const postInvokeBCItem = bo.bc[postInvoke.bc]
             const widgetName = (action.payload as any).widgetName // TODO: interface should specify widgetName
-            return Observable.of($do.bcFetchDataRequest({
-                bcName: postInvokeBCItem.name, widgetName
-            }))
+            return Observable.concat(
+                Observable.of($do.bcChangePage({bcName: postInvokeBCItem.name, page: 1})),
```

Comment:
`refreshBc` postInvoke is widely used to refresh data after custom operation which could change displayed record.  
So if the user navigated through several pages of data and then initiated custom operation he will be reverted to page 1 and will lose the record which he interacted with, which is not user-friendly.
At the same time there is a question of infiite scroll paginations, when user fetched several chunks of data with 'Load more' button, will he lose all that and only the first chunk will be displayed?
Additional info required in issue description about the problem this PR is intended to fix.

Reply:
Undo page reset for bcFetchDataRequest postaction. The bcFetchDataRequest action has been changed, which makes it possible to load pages from the first to the current, with the paginationMode parameter set.